tricerintops/ElementarySudokuSolver
# 1: Display empty sudoku puzzle when user launches app (branch to merge)

### Description
When the user launches the app, they should see an empty sudoku puzzle

<img width="300" alt="Screenshot 2021-06-12 at 14 22 29" src="https://user-images.githubusercontent.com/47641432/121784034-7a7f3280-cba9-11eb-90e6-1bfa7d5b2f01.png">

### Links
Issue: https://github.com/tricerintops/ElementarySudokuSolver/issues/1

### QA instructions
Test on several form factors
- Make sure to at least test on the smallest form factor we support (iPod Touch 7th generation or original iPhone SE)
- And the largest phone - iPhone 12 Pro Max
- Please also check on iPad!!!

Test on different orientations
- Please check portrait AND landscape for all form factors you test!!

What to look for:
- The puzzle should be centered both vertically and horizontally.
- It should almost fill the screen
